### PR TITLE
Segni.mobile responsive nav

### PR DESCRIPTION
**Pull request by Segnicho**
Screenshot 2023-04-26 8 28 59 AM
added z-index and made mobile responsive

### CODE REVIEW

File: Web/starter_project_vanguard/public/img/profilePhoto.png
Comment (Adona12) [inline Web/starter_project_vanguard/public/img/profilePhoto.png:1]: Let's use kabab case for everything other than components

File: Web/starter_project_vanguard/pages/profile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const profiles: ReactNode[] = [
+    <PersonalInfo />,
+    <></>,
+    <AccountSetting />,
+  ]
+
+  return (
+    <div className="p-10">
+      <h1 className="font-semibold text-[30px]">Profile</h1>
+      <div className="pt-6 text-sm font-medium text-center text-gray-500 border-b border-gray-200 dark:text-gray-400 dark:border-gray-700">
```
Comment (Adona12) [inline Web/starter_project_vanguard/pages/profile/index.tsx:22]: The text color can be defined globally for others to use

File: Web/starter_project_vanguard/components/profile/user.ts
Comment (Adona12) [inline Web/starter_project_vanguard/components/profile/user.ts:1]: Let's move this file in to `/data` subfolder

File: Web/starter_project_vanguard/components/layout/navbar.tsx [hunk 2]
```diff
@@ -39,11 +41,45 @@ const Navbar = () => {
...
+                currSegment === navItem.href.split('/')[1]
+                  ? 'border-b-4 border-primary text-primary'
+                  : 'text-tertiary-text'
+              )}
+              onClick={() => setToggle(!toggle)}
+            >
+              {navItem.name}
+            </Link>
+          ))}
+          <button className="bg-primary text-white rounded-md w-[78px] h-[36px] md:float-right">
```
Comment (Adona12) [inline Web/starter_project_vanguard/components/layout/navbar.tsx:78]: Let's use the globally define button styles

**Comment by Segnicho** [comment]
Screenshot 2023-04-27 8 10 37 AM

**Comment by jebessaD** [comment]
> Screenshot 2023-04-27 8 10 37 AM

The font you used has serifs. There are no serifs on the provided UI.
Please confirm that you used the same font.

File: Web/starter_project_vanguard/components/layout/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+const Navbar:React.FC = () => {
+  const [toggle, setToggle] = useState(false)
+  const [active, setActive] = useState("/")
+  const setActiveAndToggle = (current:string)=>{
+    setActive(current)
+    setToggle(!toggle)
+  }
+  return (
+    <div className="sticky top-0 z-50 bg-white bg-opacity-100 flex justify-between font-montserrat px-10 pt-5">
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/layout/Navbar.tsx:18]: [check here](https://fonts.google.com/specimen/Montserrat)
As you can see from the above link, the Montserrat font does not have serifs; I'm not sure why the font you used did. Please validate that the font you used and the UI's visual appearance are the same.

File: Web/starter_project_vanguard/components/layout/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      >
+        <AiOutlineMenu className="w-7 h-7 cursor-pointer" />
+        <div
+          className={`${
+            !toggle ? 'hidden' : 'flex'
+          } flex-col gap-4 p-6 black-gradient absolute top-12 right-2 mx-4 my-2 min-w-[140px] rounded-xl
+          z-50 bg-gray-100 bg-opacity-100
+          `}
+        >
+          {' '}
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/layout/Navbar.tsx:57]: Remove unnecessary code  `{' '}`

File: Web/starter_project_vanguard/components/layout/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+import Link from 'next/link'
+import { AiOutlineMenu } from 'react-icons/ai'
+import { navItems } from './data/nav-items'
+import { useState } from 'react'
+import logo from '../../public/img/a2sv-logo.png'
+
+
+const Navbar:React.FC = () => {
+  const [toggle, setToggle] = useState(false)
+  const [active, setActive] = useState("/")
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/layout/Navbar.tsx:12]: It is better if you can add the type.
`const [active, setActive] = useState<string>("/")` like this

**Review by jebessaD — CHANGES_REQUESTED** [review]

File: Web/starter_project_vanguard/components/layout/data/nav-items.ts
Comment (ruthgdb) [inline Web/starter_project_vanguard/components/layout/data/nav-items.ts:1]: You should move this file to data/layout folder.

**Review by ruthgdb — CHANGES_REQUESTED** [review]

File: Web/starter_project_vanguard/components/layout/data/nav-items.ts
Comment (Segnicho) [reply Web/starter_project_vanguard/components/layout/data/nav-items.ts:1]: Ok, i did

**Review by sam5235 — APPROVED** [review]

**Review by ruthgdb — APPROVED** [review]
LGTM. Well done.

**Review by jebessaD — APPROVED** [review]
LG

**Pull request merged**